lib/screen/home/home_page.dart

```diff
@@ -368,6 +395,13 @@ class HomePageState extends State<HomePage>
           (BuildContext context, int index) {
             final asset = tokens[index];
 
+            if (asset.pending == true && asset.source == 'postcard') {
```

Comment:
@ppupha looks like we didn't user `asset.source == 'postcard'` anymore. can you check that it's still working?